controlplane/kubeadm/controllers/controller_test.go

```diff
@@ -145,6 +145,9 @@ func TestReconcileReturnErrorWhenOwnerClusterIsMissing(t *testing.T) {
 	}, 10*time.Second).Should(HaveOccurred())
 }
 
+/*
+Disabling this test temporarily until we can get a fix for https://github.com/kubernetes/kubernetes/issues/80609 in controller runtim
+
 func TestReconcileUpdateObservedGeneration(t *testing.T) {
 	g := NewWithT(t)
```

Comment:
Should we use t.Skip() instead? Then we also see it in the skipped test list in Prow